src/components/DiagramFrame/DiagramFrame.tsx

```diff
@@ -30,6 +30,8 @@ import {
   IconButton,
   Layout,
 } from "@looker/components"
+import { Dashboard } from "@styled-icons/boxicons-solid"
```

Comment:
expect Dashboard from material

Reply:
This one will be removed in follow up PR